core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/SynapseTests.scala

```diff
@@ -78,4 +85,22 @@ class SynapseTests extends TestBase {
         throw t
     }
   }
+
+  test("listPythonFiles") {
+    val allPythonFiles = SynapseUtilities.listPythonFiles()
+
+    allPythonFiles.foreach(file => println(file))
+  }
+
+  test("listNoteBookFiles") {
+    val allPythonNotebooks = SynapseUtilities.listNoteBookFiles()
+
+    allPythonNotebooks.foreach(file => println(file))
+  }
+
+  test("listPythonJobFiles") {
+    val allPythonJobFiles = SynapseUtilities.listPythonJobFiles()
+
+    allPythonJobFiles.foreach(file => println(file))
+  }
```

Comment:
Are these tests that you would like to keep or something that you use for debugging? If the former we might want to add some sort of assert in here so that we can ensure they don't break. If the latter we might want to set them to "ignore" with a comment

Reply:
I think it is now fine to remove them. Thanks for catching this one!